mithril-aggregator/src/configuration.rs

```diff
@@ -52,6 +52,15 @@ pub struct Configuration {
     /// to communicate with the Cardano node
     pub cardano_node_socket_path: PathBuf,
 
+    /// Cardano node version.
+    ///
+    /// **NOTE**: This cannot be verified for now (see [this
+    /// issue](https://github.com/input-output-hk/cardano-cli/issues/224)). This
+    /// is why it has to be manually given to the Aggregator. The use of an
+    /// environment variable is enocouraged since it makes easier the update
+    /// when the cardano node is updated.
```

Comment:
Maybe second part is not needed as all the configuration parameters can all fallback on environment variables
```suggestion
    /// **NOTE**: This cannot be verified for now (see [this
    /// issue](https://github.com/input-output-hk/cardano-cli/issues/224)). This
    /// is why it has to be manually given to the Aggregator
```

Reply:
Yes I added this sentence because it can also be passed using the configuration file.

Reply:
AFAIK this is the same behavior for all the other configuration parameters. IMHO this is redundant :shrug: 
However, if you still want to keep it, there is a small type in `encouraged`.